troubleshooting/tools/tracevol.py

```diff
@@ -532,6 +540,21 @@ def get_volname_prefix(arg, pvdata):
         volname_prefix = volume_attr[key]
     return volname_prefix
 
+def get_fsname_from_pvdata(arg, pvdata):
+    """
+    Returns fsname stored in pv data
+    """
+    fsname = 'myfs'
```

Comment:
fsname is not myfs by default. if we don't find fsname we need to error out

Reply:
fixed